docs/querying/using-caching.md

```diff
@@ -0,0 +1,92 @@
+---
+id: using-caching
+title: "Using query caching"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+
+This topic covers how to configure services to populate and use the Druid query cache. For a conceptual overview and use cases, see [Query caching](./caching.md). For information on how to configure the caching mechanism, see [Cache configuration](../configuration/index.md#cache-configuration).
+
+All query caches have a pair of parameters that control the way individual queries interact with the cache:
+
+- `useCache` to instruct queries to use the cache for results.
+- `populateCache` to instruct a query to cache its results.
+
+The separation of concerns, usage and population, lets you include cached results for queries on uncommon data without polluting the cache with results that are unlikely to be reused by other queries, for example, large reports or queries on very old data.
+
+To use caching, you must first enable the settings for a service in the runtime properties. Afterward, you can control which queries use the cache or populate the cache on a per-query basis within the query context.
+
+The query context settings do not override the server settings. For example, it is possible to set a query context to use the cache, but if it is not enabled at the service level, there's no cache to use.
```

Comment:
I think this is a little more nuanced. It doesn't enable the feature if it's disabled in the server settings, but a query can specify that it doesn't want to populate the cache or use the cache, even if those settings are made in the server settings

Reply:
I just removed this. I think the sentence in the previous paragraph is pretty clear.